synthetic-dataset/make_set.py

```diff
@@ -46,7 +46,8 @@ def make_one_set(smoke_video_file, background_file, set_idx, fx=0.3,
         dx = random.randint(0, wbg - ws - 1)
 
         for i, (img, smoke) in enumerate(zip(imgs, smoke_imgs)):
-            result, mask = basic_blending(img, smoke, offset=(dy, dx))
+            # result, mask = basic_blending(img, smoke, offset=(dy, dx))
+            result, mask = poisson_blending(img, smoke, offset=(dy, dx))
```

Comment:
To be added in the dictionary after this PR is merged : https://github.com/pyronear/synthetic-dataset/pull/4

Reply:
done :)